docs-mslearn/toolkit/hubs/private-networking.md

```diff
@@ -74,6 +74,67 @@ Before enabling private access, review the networking details on this page to un
 
 <br>
 
+## Removing private networking
+
+If you need to reduce costs or simplify your FinOps hub deployment, you can remove private networking and switch back to public access. This change will:
+
+- Remove the virtual network and associated networking costs
+- Disable private endpoints and DNS zones
+- Configure storage, Data Explorer, and Key Vault to use public access
+- Switch Azure Data Factory back to the public integration runtime
+
+> [!WARNING]
+> Removing private networking is a significant change that will affect how you access your FinOps hub. Ensure all stakeholders understand the security implications before proceeding.
+
+### Steps to remove private networking
+
+1. **Plan the transition**:
+   - Identify all users and systems currently accessing the hub via private networking
+   - Coordinate with your network administrators about the change
+   - Schedule maintenance window as the hub will be temporarily inaccessible during the transition
+
+2. **Update the FinOps hub deployment**:
+   - Navigate to your FinOps hub resource group in the Azure portal
+   - Locate the deployment template or redeploy using the original template
+   - Set **Access** to **Public** on the **Advanced** tab
+   - Review all other settings to ensure they remain as desired
+   - Deploy the updated configuration
```

Comment:
There are 2 options - 
1) redeploy the current version by using the deployments tab on the resource group - open the original deployment and choose redeploy. Then pick public.
2) install the latest current version of the toolkit using the same resource group, hub and adx name (which can also be obtained from the template or the config.json file)